DashAI/back/metrics/Classification/f1score.py

```diff
@@ -0,0 +1,48 @@
+import numpy as np
+from sklearn.metrics import f1_score
+
+from DashAI.back.metrics.classification_metric import ClassificationMetric
+
+
+class F1Score(ClassificationMetric):
```

Comment:
`F1` or `F1Score` ?

The following metrics does not have that suffix.
I think it is good to standardize this at this point.